show/interfaces/__init__.py

```diff
@@ -639,3 +639,36 @@ def autoneg_status(interfacename, namespace, display, verbose):
         cmd += " -n {}".format(namespace)
 
     clicommon.run_command(cmd, display_cmd=verbose)
+
+#
+# link-training group (show interfaces link-training ...)
```

Comment:
>show interfaces [](http://example.com/codeflow?start=23&length=15)

Is there a reason to choose `show interfaces` vs `show interface`?

Reply:
I have no idea, this is a legacy command, and my best guess is that it supports both signle interface and multi-interface at the same time, hence the original author adopted 'show interfaces' instead of 'show interface'

Reply:
Could you fix the PR description? I see many wrong subcommand.
```
sudo config interface link-training Ethernet0 on
```

Reply:
Sorry, I'm not following you, is this about 'interfaces' v.s. 'interface' in show and config command?
If that's the case, this is a legacy command format since day 1, while the link-training is actually a plug-in to these legacy CLICK based CLI commands, hence it's not expected to change the upper level command syntax.

Reply:
Got it. I see the difference between `show` and `config`.